src/Component/Page/Home.purs

```diff
@@ -0,0 +1,208 @@
+module Component.Page.Home where
+
+import Prelude
+
+import Api.Endpoint (ArticleParams, noArticleParams)
+import AppM (Env)
+import Capability.LogMessages (class LogMessages)
+import Capability.ManageResource (class ManageResource, getArticles, getTags)
+import Capability.Navigate (class Navigate, navigate)
+import Capability.Now (class Now)
+import Component.Classes as CC
+import Component.HTML.ArticleList (articleList)
+import Component.HTML.Footer (footer)
+import Component.HTML.Header (header)
+import Control.Monad.Reader (class MonadAsk)
+import Data.Article (Article)
+import Data.Const (Const)
+import Data.Either (either)
+import Data.Maybe (Maybe(..))
+import Data.Route (Route(..))
+import Effect.Aff.Class (class MonadAff)
+import Halogen as H
+import Halogen.HTML as HH
+import Halogen.HTML.Events as HE
+import Halogen.HTML.Properties as HP
+import Halogen.Themes.Bootstrap4 as HB
+import Network.RemoteData (RemoteData(..))
+import Web.Event.Event (Event, preventDefault)
+import Web.UIEvent.MouseEvent.EventTypes (click)
+
+type State =
+  { tags :: RemoteData String (Array String)
+  , articles :: RemoteData String (Array Article)
+  , tab :: Tab
+  }
+
+data Tab
+  = FeedTab
+  | GlobalTab
+  | TagTab String
+
+data Query a
+  = Init a
+  | LoadTags a
+  | Navigate Route a
+  | ShowTab Tab Event a
+  | LoadArticles ArticleParams a
+
+component
+  :: forall m
+   . MonadAff m
+  => MonadAsk Env m
+  => Now m
+  => LogMessages m
+  => ManageResource m
+  => Navigate m
+  => H.Component HH.HTML Query Unit Void m
+component =
+  H.lifecycleParentComponent
+    { initialState: const { tags: NotAsked, articles: NotAsked, tab: GlobalTab } 
+    , render
+    , eval
+    , receiver: const Nothing
+    , initializer: Just $ Init unit
+    , finalizer: Nothing
+    }
+
+  where 
+
+  eval :: Query ~> H.ParentDSL State Query (Const Void) Void Void m
+  eval = case _ of
+    Init a -> do
```

Comment:
It's a nice idea to fork here rather than delay page load. However, I'd go ahead and fork both of those queries directly.

```purescript
do
  void $ H.fork $ eval $ LoadTags a
  void $ H.fork $ eval $ LoadArticles noArticleParams a
  pure a
```